.github/workflows/ci.yml

```diff
@@ -88,3 +88,42 @@ jobs:
       - "lint"
       - "build"
       - "pylint"
+  publish_gh:
+    name: "Publish to GitHub"
+    runs-on: "ubuntu-20.04"
```

Comment:
we used `"ubuntu-latest"` in the build job, maybe we should make it consistent

Reply:
Yes we should, and we should keep it at `ubuntu-20.04`. I'll double check my copy/pasting that it is the same across the board of 20.04